electrical/readme.MD

```diff
@@ -0,0 +1,261 @@
+<!--[title](Pictures/Electronics/electronics title.png)
```

Comment:
This file should be renamed to reflect the original organization: electrical_build.md

Reply:
done